src/Components/MixedType/MixedType.js

```diff
@@ -1,10 +1,49 @@
-import React, {useState} from 'react';
-import { useForm } from 'react-hook-form';
+import React from 'react';
 import './MixedType.scss';
 
-function MixedType() { 
-    return(
-        <div></div>
+const MixedType=({ qno, data, id, isEdit = false, onUpdate})=> { 
+    const options = ['Very satisfied','Satisfied', 'Neutral', 'Dissatisfied','Very dissatisfied'];
```

Comment:
No hard coded value in child component, it needs to blindly accept the values from parent and use it dynamically